usr/libexec/greenboot/greenboot.sh

```diff
@@ -2,75 +2,56 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-run_required_scripts () {
-  echo "Running Required Health Check Scripts..."
-  local required_scripts=`find /etc/greenboot/check/required.d -name '*.sh'`
+script_runner () {
+  if [ $# -eq 0 ]; then
+    echo "Please call '$0 run <dir_path> [{strict,relaxed}] [message]' to run this command!"
+    exit 1
+  fi
   local rc=0
-  for script in $required_scripts; do
+  local scripts=`find $1 -name '*.sh'`
+  case "${2:-s}" in
```

Comment:
Honestly, it's probably better to just let callers spell it out the long way (`strict` or `relaxed`) to make it more self-documenting.

Reply:
ack